### PR TITLE
Documentation for Route Configuration

### PR DESCRIPTION
**Pull request by MikeRyan52**
Added basic guides for working with route configuration

### CODE REVIEW

File: docs/getting-started/redirect.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  {
+    path: '/',
+    component: HomePage
+  },
+  {
+    path: '/blog',
+    component: BlogPage,
+    children: [
+      {
+        path: ':id',
```
Comment (brandonroberts) [inline docs/getting-started/redirect.md]: Missing some slashes here

File: docs/getting-started/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Getting Started
+To get started, first install the ngrx/router using npm:
+```
+npm install @ngrx/router
+```
+
```
Comment (brandonroberts) [inline docs/getting-started/README.md]: This numbered list doesn't show on separate lines

File: docs/getting-started/README.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Getting Started
+To get started, first install the ngrx/router using npm:
+```
```
Comment (brandonroberts) [inline docs/getting-started/README.md]: Add `--save` here

File: docs/getting-started/code-splitting.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
+# Code Splitting with Asynchronous Routes
```
Comment (brandonroberts) [inline docs/getting-started/code-splitting.md]: Might be nice to have example for Webpack and SystemJS here

File: docs/getting-started/index-route.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+const routes: Routes = [
+  {
+    path: '/blog',
+    component: BlogPage
+    indexRoute: {
+      component: BlogPostsPage
+    },
+    children: [
+      {
+        path: ':id',
```
Comment (brandonroberts) [inline docs/getting-started/index-route.md]: Missing a slash

Comment (MikeRyan52) [inline docs/getting-started/index-route.md]: Nope, no slashes for children

**Comment by brandonroberts** [comment]
Is this one ready now?

**Comment by MikeRyan52** [comment]
Yes, it is ready for review

**Comment by brandonroberts** [comment]
LGTM

**Pull request merged**